tests/demo_tests/test_ip_v4v6_device_raw_capture.py

```diff
@@ -0,0 +1,244 @@
+import pytest
+import snappi
+
+
+@pytest.fixture
+def app_intf(utils):
+    api = snappi.api(host=utils.settings.api_server, ext=utils.settings.ext)
+    yield api
+    if getattr(api, 'assistant', None) is not None:
+        api.assistant.Session.remove()
+
+
```

Comment:
Most comments in tests/demo_tests/test_ip_v4v6_raw_capture.py apply here as well.

One extra thing, add a TODO for checking protocol summar (currently not supported).

Reply:
addressed